pkgs/by-name/ca/cargo-tauri/test-app.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  cargo-tauri,
+  darwin,
+  glib-networking,
+  libsoup,
+  openssl,
+  pkg-config,
+  webkitgtk,
+  wrapGAppsHook3,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "test-app";
+  inherit (cargo-tauri) version src;
+
+  # Basic example provided by upstream
+  sourceRoot = "${src.name}/examples/workspace";
+
+  cargoPatches = [
+    # https://github.com/NixOS/nixpkgs/issues/332957
+    ./update-time-crate.patch
+  ];
```

Comment:
Could you generate this patch by running `cargo update time` instead? That should reduce the size of the patch I think